src/blueapi/service/runner.py

```diff
@@ -32,51 +35,52 @@ class WorkerDispatcher:
     _state: EnvironmentResponse
 
     def __init__(
-        self, config: ApplicationConfig | None = None, use_subprocess: bool = True
+        self,
+        config: ApplicationConfig | None = None,
+        use_subprocess: bool = True,
     ) -> None:
         self._config = config or ApplicationConfig()
         self._subprocess = None
         self._use_subprocess = use_subprocess
-        self._state = EnvironmentResponse(initialized=False)
+        self._state = EnvironmentResponse(
+            initialized=False,
+        )
+
+    def reload(self):
```

Comment:
Could: There is a functionality equivalent implementation in `main.py`, should probably just call this.

Reply:
Done